privaterelay/templates/faq.html

```diff
@@ -6,65 +6,58 @@
-        </h2>
-        <p class="faq-answer">
-          We will provide users with advance notice that they will need to change the email address of any of their accounts that are using relay addresses.
-        </p>
-      </section>
+      <div class="faq-col flx flx-col">
+        <section class="faq">
+          <h2 class="faq-headline">How can I join the private beta?</h2>
+          <p class="faq-answer">
+            First, you’ll need a Firefox account. Then, use your account to sign up for the waiting list. When a spot opens, we’ll send you an email with a link you can use to complete registration and start using Relay.
+          </p>
+        </section>
+        <section class="faq">
+          <h2 class="faq-headline">What about spam?</h2>
+          <p class="faq-answer">
+            While Relay does not filter for spam, our email partner Amazon SES does block spam and malware. If Relay forwards messages you don’t want, you can update your Relay settings to block messages from the alias forwarding them.
+          </p>
+          <p class="faq-answer">
+            If you see a broader problem of unwanted email from all of your aliases, please report this to us so we can consider adjusting the SES spam thresholds for this service.
+          </p>
+          <p class="faq-answer">
+            If you report these as spam, your email provider will see Relay as the source of spam, not the original sender.
+          </p>
+        </section>
+        <section class="faq">
+          <h2 class="faq-headline">Why won’t a site accept my Relay alias?</h2>
+          <p class="faq-answer">
+            Some sites may not accept an email address that includes a subdomain (ie, the “relay” portion of @relay.firefox.com) and others have stopped accepting all addresses except those from Gmail, Hotmail, or Yahoo accounts. As Firefox Relay grows in popularity and issues more aliases, our service might be placed on a blocklist. If you are not able to use a Relay alias, please let us know.
+          </p>
+        </section>
+      </div>
+      <div class="faq-col flx flx-col">
+        <section class="faq">
+          <h2 class="faq-headline">Is Relay available only in the US?</h2>
+          <p class="faq-answer">
+            The site is currently only available in English, but you can use the service anywhere.
+          </p>
+        </section>
+        <section class="faq">
+          <h2 class="faq-headline">Can I reply to messages using my Relay alias?</h2>
+          <p class="faq-answer">
+            Relay does not yet offer the ability to reply using an alias. If you try, nothing will happen. We are working on additional features to let you <a href="https://github.com/mozilla/fx-private-relay/issues/151" rel="noopener noreferrer" class="text-link">send an automated response</a> or <a href="https://github.com/mozilla/fx-private-relay/issues/99" rel="noopener noreferrer" class="text-link">reply anonymously to senders</a>.
```

Comment:
We're only planning the anonymous reply feature. The other feature mentioned here isn't accurate. (We might try to reply back to relay users *WHEN THEY TRY TO REPLY TO ORIGINAL SENDERS* - but we don't plan to build a feature for relay users to send automatic replies.)

```suggestion
            Relay does not yet offer the ability to reply using an alias. If you try, nothing will happen. We are planning an additional feature to let you <a href="https://github.com/mozilla/fx-private-relay/issues/99" rel="noopener noreferrer" class="text-link">reply anonymously to senders</a>.
```